package.json

```diff
@@ -29,11 +29,14 @@
     "typescript": "^4.5.2"
   },
   "scripts": {
+    "type-check": "tsc",
     "lint": "eslint packages/**",
-    "check": "prettier packages/** --check && tsc && npm run lint && npm run build && npm run test",
+    "format-check": "prettier packages/** --check",
+    "check": "npm run format-check && npm run type-check && npm run lint && npm run build && npm run test",
     "prettify": "prettier packages/** --write",
     "build": "npm run build --workspace=wrangler",
-    "test": "npm run test --workspace=wrangler"
+    "test": "npm run test --workspaces --if-present",
+    "changeset": "changeset"
```

Comment:
oh you did actually add the scripts here.

Reply:
Yeah.
One problem with `npm` workspaces is that it is hard to delegate from a workspace up to the root when you run a script.

I was thinking of adding copies of these scripts to the `wrangler` workspace that worked, but then I was concerned that we would need to remember to keep everything up to date... 🤔

Encouraging all scripts to be run from the root made things simpler, yet more complicated...